pkg/common/constants.go

```diff
@@ -29,6 +29,9 @@ const LabelApplicationID = "applicationId"
 const LabelQueueName = "queue"
 const ApplicationDefaultQueue = "root.sandbox"
 const DefaultPartition = "default"
+const AppTagNamespace = "namespace"
+const AppTagNamespaceResourceQuota = "namespace.resourcequota"
```

Comment:
These two should be moved to the SI as we use them in both the core and shim

Reply:
Yes, I am adding this in another PR.